File: pset6/mario/mario.py
from cs50 import get_int



while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

for i in range(h):
        print(" " * (h - i - 1), end="")
        print("#" * (i + 1), end="")
        print("  ", end="")
        print("#" * (i + 1))

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        //ask height or number of rows
        n = get_int("height?: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        //left create reverse number of dots first
        for (int j = n; j > i + 1; j--)
        {
            printf(" ");
        }

        //left create number of #
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("  ");
        //right create number of #
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}